src/Compilers/CSharp/Test/Emit3/Semantics/ExtensionTests.cs

```diff
@@ -109,6 +109,17 @@ extends [netstandard]System.Object
     .class nested public auto ansi sealed beforefieldinit '<>E__0'
         extends [netstandard]System.Object
     {
+        // Methods
+        .method public hidebysig specialname static 
+            void '<Extension>$' (
+                object ''
+            ) cil managed 
+        {
+            // Method begins at RVA 0x2067
+            // Code size 1 (0x1)
+            .maxstack 8
+            IL_0000: ret
+        } // end of method '<>E__0'::'<Extension>$'
```

Comment:
Consider adding a LangVer test (you can consume extension methods from metadata from any language version)

Reply:
> Consider adding a LangVer test (you can consume extension methods from metadata from any language version)

1. I think the behavior should be different, the suggested behavior goes against the spirit of the language version feature, I think.
2. This level of testing is outside of scope of the PR. It is not about binding call-sites. While the language check should be performed for a call-site during binding.